lib/time_picker/ddd_time_picker.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:ddd_base_project/design_system/widget/app_theme.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DDDTimePicker extends StatefulWidget {
+  final DateTime? initialTime;
+
+  const DDDTimePicker({
+    Key? key,
+    this.initialTime,
+  }) : super(key: key);
+
+  @override
+  State<DDDTimePicker> createState() => _DDDTimePickerState();
+}
+
+class _DDDTimePickerState extends State<DDDTimePicker> {
+  late DateTime _selectedTime;
+
+  @override
+  void initState() {
+    super.initState();
+    _selectedTime = widget.initialTime ??
```

Comment:
just make _selectedTime optional so you don't need to give it custom constructor date. Cupertino picker will take current date